subprojects/composite-builds/src/test/groovy/org/gradle/composite/internal/DefaultIncludedBuildRegistryTest.groovy

```diff
@@ -65,14 +64,7 @@ class DefaultIncludedBuildRegistryTest extends Specification {
     def services = new DefaultServiceRegistry()
     def modelServices = Mock(BuildModelControllerServices)
     def buildTree = Mock(BuildTreeState)
-    def factory = new BuildStateFactory(
-        buildTree,
-        listenerManager,
-        Stub(GradleEnterprisePluginManager),
-        Stub(GradleUserHomeScopeServiceRegistry),
-        Stub(CrossBuildSessionState),
-        Stub(BuildCancellationToken)
-    )
+    def factory = new BuildStateFactory(buildTree, listenerManager, Stub(GradleUserHomeScopeServiceRegistry), Stub(CrossBuildSessionState), Stub(BuildCancellationToken))
```

Comment:
💄 Please keep the multiline format for very long lines like this one as it makes it easier to review the code.